community/contributing/docs/github-changes.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "Make a Change Using the GitHub UI"
```

Comment:
GitHub Web UI, right? Cause there's also a desktop client w/ a UI

Reply:
I updated the title and .yml nav to reflect this change